orchestrator/services/executors/celery.py

```diff
@@ -80,7 +67,7 @@ def _celery_resume_process(
     user_inputs: list[State] | None = None,
     user: str | None = None,
     **kwargs: Any,
-) -> UUID:
+) -> bool:
```

Comment:
Now that we're revisiting things; this function calls:
`store_input_state(pstat.process_id, user_inputs, "user_input")`

Could you move this to `resume_process()` in the `processes.py` file? I think it should be done there after the `post_form` validation, with a `if user_inputs:` check to prevent inserting empty input.

This would make it consistent with how it's done for new processes (`start_process()` calls `create_process()` which then does `store_input_state(process_id, state | initial_state, "initial_state")`).